.github/workflows/release.yml

```diff
@@ -7,17 +7,31 @@ on:
   workflow_dispatch:
 
 jobs:
+  version:
+    outputs:
+      version: ${{ steps.calculate_version.outputs.version }}
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+      with:
+        submodules: recursive
```

Comment:
Can we avoid loading the submodules for this step? It can save us a bit of CI time.